submissions/Zakkarat/index.js

```diff
@@ -0,0 +1,92 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+const INHABITANT = {
+    species: {
+        human: 'human'
+    },
+    genders: {
+        male: 'male',
+        female: 'female'
+    }
+};
+const Character = function (name, gender, legs, saying) {
+  this.name = name;
+  this.gender = gender;
+  this.legs = legs;
+  this.saying = saying;
+  }
+  const Cat = function (name, gender, legs, saying) {
+    Character.call(this, name, gender, legs, saying);
+    this.species = "cat";
+  }
+  const Dog = function (name, gender, legs, saying) {
+    Character.call(this, name, gender, legs, saying);
+    this.species = "dog";
+  }
+  Character.prototype.sentence = function() {
+    let sentence = [
+        `Hey there, my name is <strong>${this.name}</strong>`,
+        `and I am a ${this.species}!`,
+        `My gender is ${this.gender},`,
+        `in addition I have ${this.legs} legs`,
+        `and ${this.hands} hands.`,
+        `Wait a minute... ${this.saying}`
+    ]
+    if(this.hands == null){
```

Comment:
Do you know what objects evaluate to `false` in conditional statements in JS? :)

Reply:
Thank you! Done in proper way!